contributors/devel/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
Should probably be sig contribx except for a few things like api which are sig architecture